category/internblogs/index.html

```diff
@@ -0,0 +1,3 @@
+---
+category: internblogs
+---
```

Comment:
Add a new page for summer2016 as well. Lookup `coverstory` or `bigstory` for reference.